clients/unshaded/src/test/java/tachyon/client/file/options/CreateOptionsTest.java

```diff
@@ -60,7 +54,7 @@ public void defaultsTest() {
     UnderStorageType ufsType = UnderStorageType.SYNC_PERSIST;
     conf.set(Constants.USER_BLOCK_SIZE_BYTES_DEFAULT, "64MB");
     conf.set(Constants.USER_FILE_UNDER_STORAGE_TYPE_DEFAULT, ufsType.toString());
-    Whitebox.setInternalState(ClientContext.class, "sTachyonConf", conf);
+    ClientContext.reset(conf);
```

Comment:
Since you're on this, I think `reset` is not ideal for mocking, because the change here is not isolated and therefore can affect other test cases. Maybe we can use factory to get ClientContext, and we can inject a `MockedClientContext` to it?